.github/actions/setup-builder/action.yaml

```diff
@@ -34,6 +34,7 @@ runs:
       run: |
         apt-get update
         apt-get install -y protobuf-compiler
+        apt-get install -y clang
```

Comment:
So we did not have clang before?

Reply:
I think we dont' as `fs-hdfs` failed before that it was not able to find `libclang*.so` files